### DISCUSSION TITLE
🚀 Implementing "titleBarStyle: hiddenInset"

### DISCUSSION THREAD

**Discussion by mazwrld** [zed-industries/zed#8311] [Feature Requests]
Hey everyone,

I'm looking to implement a hidden inset title bar style similar to Electron's titleBarStyle: "hiddenInset". While this feature isn't crucial, I believe it could enhance the user experience of Zed. 

Something like this.
![CleanShot 2024-02-23 at 16 23 59](https://github.com/zed-industries/zed/assets/34517221/ed848012-664a-4742-989e-b6a63cb89539)

Does anyone else see see the value in this?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
